src/cassandra.go

```diff
@@ -52,14 +52,12 @@ func main() {
 		rawMetrics, allColumnFamilies, err := getMetrics()
 		fatalIfErr(err)
 
-		hostnameAttr := metric.Attr("hostname", args.Hostname)
-		portAttr := metric.Attr("port", strconv.Itoa(args.Port))
-		ms := e.NewMetricSet("CassandraSample", hostnameAttr, portAttr)
+		ms := metricSet(e, "CassandraSample")
 		populateMetrics(ms, rawMetrics, metricsDefinition)
 		populateMetrics(ms, rawMetrics, commonDefinition)
 
 		for _, columnFamilyMetrics := range allColumnFamilies {
-			s := e.NewMetricSet("CassandraColumnFamilySample")
```

Comment:
this was actually a bug 😱 . We were not adding `host` and `port` for this metric.

Anyway, but these changes it's fixed now 👍